readme.rst

```diff
@@ -258,6 +258,26 @@ Add redirects
 You can add redirects to make sure existing links and bookmarks continue working when you move files around.
 To do so, specify the old and new paths in the ``redirects`` setting of the ``custom_conf.py`` file.
 
+Configure included extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The starter pack includes a set of extensions that we deem useful for all documentation sets.
```

Comment:
"that the Technical Authors deem" ?

or maybe just

"includes a recommended set of extensions."